src/MustardBlack.Html.Forms/Components/DropDownComponent.cs

```diff
@@ -85,10 +85,10 @@ public virtual IDropDownComponent WithoutNullOption()
 
 		protected override string RenderComponent()
 		{
-			var builder = new TagBuilder("select", this.htmlAttributes);
-
 			if (this.ariaLabel)
-				this.htmlAttributes.Add("aria-label", this.termResolver.ResolveTerm(this.Label, this.culture));
+				this.htmlAttributes["aria-label"] = this.termResolver.ResolveTerm(this.Label, this.culture);

```

Comment:
How come we changed this to use `this.htmlAttributes["aria-label"]`? Can we guarantee that the `aria-label` is already in the dictionary?

Reply:
This syntax is safer
```
var dic = new Dictionary<string,string>();
dic["hello"] = "hi";
dic["hello"] = "hi2";
```

Will create a key "hello" with the value "hi" then overwrite with hi2

```
var dic = new Dictionary<string,string>();
dic.add("hello","hi");
dic.add("hello","hi2");
```
Will create a key "hello" with the value "hi" then throw and error because the key exists

Reply:
Nice. TIL. I assumed the first syntax would throw if it didn't exist.